minet/bluesky/http_client.py

```diff
@@ -101,30 +110,136 @@ def request(
             headers=headers,
         )
 
+        if response.status < 200 or response.status >= 300:
+            data = response.json()
+            if "error" in data:
+                e = data["error"]
+                if response.status == 400:
+                    if e == "UpstreamFailure":
+                        raise BlueskyUpstreamFailureError(
+                            "Bluesky is currently experiencing upstream issues."
+                        )
+                    elif e in (
```

Comment:
This is not needed per se. A `400` status is always a variant of "given parameters" are not valid. This should crash the process without retries (as is the case already don't worry) since it means an error on our end usually.